pkg/controllers/edgeconnect/controller.go

```diff
@@ -178,13 +178,13 @@ func (controller *Controller) reconcileEdgeConnectDeletion(ctx context.Context,
 	return edgeConnectClient.DeleteEdgeConnect(tenantEdgeConnect.ID)
 }
 
-func (controller *Controller) deleteConnectionSetting(edgeConnectClient edgeconnect.Client, edgeConnect *edgeconnectv1alpha1.EdgeConnect) error {
+func (controller *Controller) deleteConnectionSetting(edgeConnectClient ecclient.Client, edgeConnect *edgeconnect.EdgeConnect) error {
```

Comment:
```suggestion
func (controller *Controller) deleteConnectionSetting(edgeConnectClient ecclient.Client, ec *edgeconnect.EdgeConnect) error {
```

The same applies for all other occasions in this (and other) files.

Reply:
fixed in fixed in 3f1de959